README.md

```diff
@@ -1,42 +1,80 @@
-* Javascript [Github](https://github.com/splitio/javascript-client) [Docs](http://docs.split.io/docs/javascript-sdk-overview)
-* Node [Github](https://github.com/splitio/javascript-client) [Docs](http://docs.split.io/docs/nodejs-sdk-overview)
-* .NET [Github](https://github.com/splitio/.net-core-client) [Docs](http://docs.split.io/docs/net-sdk-overview)
-* Ruby [Github](https://github.com/splitio/ruby-client) [Docs](http://docs.split.io/docs/ruby-sdk-overview)
-* PHP [Github](https://github.com/splitio/php-client) [Docs](http://docs.split.io/docs/php-sdk-overview)
-* Python [Github](https://github.com/splitio/python-client) [Docs](http://docs.split.io/docs/python-sdk-overview)
-* GO [Github](https://github.com/splitio/go-client) [Docs](http://docs.split.io/docs/go-sdk-overview)
-* Android [Github](https://github.com/splitio/android-client) [Docs](https://docs.split.io/docs/android-sdk-overview)
-* IOS [Github](https://github.com/splitio/ios-client) [Docs](https://docs.split.io/docs/ios-sdk-overview)
-
-For a comprehensive list of opensource projects visit our [Github page](https://github.com/splitio?utf8=%E2%9C%93&query=%20only%3Apublic%20).
-
-**Learn more about Split:** 
-
-Visit [split.io/product](https://www.split.io/product) for an overview of Split, or visit our documentation at [docs.split.io](http://docs.split.io) for more detailed information.
-
-**System Status:**
-
-We use a status page to monitor the availability of Split’s various services. You can check the current status at [status.split.io](http://status.split.io).
+# Split .NET SDK
+
+![CI Status](https://ci.appveyor.com/api/projects/status/github/splitio/.net-core-client?branch=master&svg=true&passingText=master%20-%20ok&pendingText=master%20-%20running&failingText=master%20-%20failing)
+
+## Overview
+This SDK is designed to work with Split, the platform for controlled rollouts, which serves features to your users via a Split feature flag to manage your complete customer experience.
+ 
+[![Twitter Follow](https://img.shields.io/twitter/follow/splitsoftware.svg?style=social&label=Follow&maxAge=1529000)](https://twitter.com/intent/follow?screen_name=splitsoftware)
+ 
+## Compatibility
+This SDK is compatible with NET Framework 4.0 and above and NET Core 1.x, 2.x and 3.0.
+
+## Getting started
+Below is a simple example that describes the instantiation and most basic usage of our SDK:
+
+```c#
+var config = new ConfigurationOptions();
+var factory = new SplitFactory("API_KEY", config);
+var sdk = factory.Client();
+
+try
+{
+    sdk.BlockUntilReady(10000);
+}
+catch (Exception ex)
+{
+    // log & handle 
+}
+
+var treatment = sdk.GetTreatment("CUSTOMER_ID","SPLIT_NAME");
+
+if (treatment == "on") 
+{
+    // insert code here to show on treatment
+} 
+else if (treatment == "off") 
+{
+    // insert code here to show off treatment 
+} else 
+{
+    // insert your control treatment code here
+}
```

Comment:
Is the indentation here correct and consistent with how it should be? I see all curly braces in their own line, but line 39 has the else too.